husky_gazebo/launch/empty_world.launch

```diff
@@ -2,7 +2,7 @@
 <!--
 Software License Agreement (BSD)
 
-\file      husky_empty_world.launch
+\file      empty_world.launch
 \authors   Paul Bovbel <pbovbel@clearpathrobotics.com, Devon Ash <dash@clearpathrobotics.com>
```

Comment:
Neither of these people have worked at CPR for many years. Should consider mass-updating at some point.